docs/multipass.md

```diff
@@ -0,0 +1,43 @@
+# Testing LibreTime with Canonical's Multipass
+
+Ever wanted to test out LibreTime but didn't want to tango with
+Virturalbox, Vagrent, or Docker? Canonical released [Multipass](https://multipass.run), a tool for setting up Ubuntu VMs with cloud-init files in a snap.
+Multipass is available for Windows and macOS, as well as Linux OSes that support snaps.
+
+Similar to Docker, Multipass works through a CLI. To use, clone this repo and then open a Terminal
+(or Command Prompt) inside the `libretime` folder and run
+```
+multipass launch bionic -n ltTEST --cloud-init libretimeTest.yaml  # to launch VM
+multipass shell ltTEST  # to enter VM's shell
+```
+
+Due to a known bug, Multipass is not able to perform the entire install using the cloud-init file.
```

Comment:
Is this bug listed anywhere accessible? Maybe link to it here, or mention that it is a Multipass bug, not a LT bug

Reply:
Anything here?

Reply:
I think I'm going to remove the Multipass docs altogether until Canonical fixes the bug.